src/components/ImageWithSideTitle/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import cn from 'classnames';
+
+export const TITLE_COLOR = {
+  BLUE: 'blue',
+  RED: 'red',
+  LIGHTBLACK: 'lightblack',
+  DAWNBLACK: 'dawnblack',
+  CLOUDGREY: 'cloudgrey',
+  SUBMARINEGREY: 'submarinegrey',
+  WHITEGREY: 'whitegray',
+  SILVERGREY: 'silvergrey',
+  OFFWHITE: 'offwhite',
+  GREEN: 'green',
+  WHITE: 'white',
+  BLACK: 'black'
+};
+
+const ImageWithSideText = ({ imgUrl, title, titleColor, subTitle }) => {
+  return (
+    <div className="flex justify-around rounded py-2">
+      <img src={imgUrl} alt="" />
```

Comment:
- should have space between the image and the title ( put the whole div in a button to see what I mean  )
- this image should be an icon that can change its color by a prop 
- and should test it like the post example

**Give me an example by storybook like this format**
![Screenshot from 2020-10-05 18-54-33](https://user-images.githubusercontent.com/46047009/95109107-69d60d00-073c-11eb-9783-31df0124c89e.png)